charts/osm/crds/meshconfig.yaml

```diff
@@ -91,7 +91,7 @@ spec:
                       description: Global list of ports to exclude from outbound traffic interception by the sidecar proxy.
                       type: array
                       items:
-                        type: string
+                        type: integer
                         pattern: ^[1-9]\d*$
```

Comment:
integer does not have a pattern I think?

Reply:
it is allowed, and I would want a check for positive integers only